swagger.yaml

```diff
@@ -3150,6 +3150,7 @@ definitions:
       - api-vip-connectivity-check
       - ntp-synchronizer
       - fio-perf-check
+      - image-availability
```

Comment:
```suggestion
      - machine-config-image-availability
```
Is this better?  I had no idea what image this was referring to until I looked at the JIRA ticket.

Reply:
@avishayt I made it generic on purpose. We would be able to use it for other images except for the MCO image.
For example, the controller image. Or other images if they will be.
So the API was made to pass an array of images.

Reply:
So at least container-image-availability?  To avoid confusion between this and boot ISOs for example.